src/model/formulations/ArrCon2000/ramp.jl

```diff
@@ -2,6 +2,26 @@
 # Copyright (C) 2020, UChicago Argonne, LLC. All rights reserved.
 # Released under the modified BSD license. See COPYING.md for more details.
 
+"""
+    _add_ramp_eqs!
+
+Ensure constraints on ramping are met.
+Based on Arroyo and Conejo (2000).
+Eqns. (24), (25) in Kneuven et al. (2020).
+
+Variables
+---
+* :is_on
+* :switch_off
+* :switch_on
+* :prod_above
+* :reserve
```

Comment:
Just a note, but this could be simultaneously documented and enforced by Julia itself by including `Gar1962.ProdVars` and `Gar1962.StatusVars` in the signature of the function.

Reply:
I see. How can I tell what those vars are?

Reply:
Should I add that documentation to https://github.com/ANL-CEEESA/UnitCommitment.jl/blob/a7f9e84c317d5caf9ba64d3168f40e238aa545bf/src/model/formulations/Gar1962/structs.jl#L20-L21

Reply:
Yes, that's the idea. `Gar1962.ProdVars` could have a docstring explaining what variables are available and what do they mean. Any method that uses Gar1962 variables should include it in the method signature. This is not only for documentation, but it would allow us to create an alternative version of the method which works for a different set of decision variables.

Reply:
Makes sense. I'll make those changes.